templates/default/aurora-security.erb

```diff
@@ -0,0 +1,7 @@
+[users]
+<% @admins.sort.each do |key, value| %>
+<%= key %> = <%= value %>, admin
+<% end %>
+
+[roles]
+admin = *
```

Comment:
Yes, customizing roles might be interesting. However, we would have to update the structure of user related information to add the groups each user belong to. This might be done in another commit.